docker/Dockerfile

```diff
@@ -1,19 +1,9 @@
-FROM ethereum/client-go:v1.12.0
-
-RUN apk update
-RUN apk add --no-cache nginx
-
-RUN nginx -v
```

Comment:
why we don't need nginx now? is this what the latest upstream code is?

Reply:
I missing it and will keep same with the upstream code.